package.json

```diff
@@ -29,7 +29,7 @@
   },
   "homepage": "https://github.com/rdfjs-base/data-model",
   "dependencies": {
-    "@types/rdf-js": "^2.0.1"
+    "@types/rdf-js": "*"
```

Comment:
Is there a reason why this dependency is relaxed to `*` instead of `^4`?

Reply:
A `^4` range excludes future major updates when that package becomes `5.0`. This can cause npm to install duplicate versions of the typings package leading to compile issues from TypeScript, especially in monorepo scenarios

Reply:
I'm confused: compile errors is exactly what one would want if the typings are incompatible, no?

Reply:
It's false negatives I'm talking about. Not really about incompatible types. The error looks similar to

> src/index.ts(3,14): error TS2742: The inferred type of 'NamedNode' cannot be named without a reference to '../packages/in-monorepo/node_modules/rdf-js'. This is likely not portable. A type annotation is necessary.